### PR TITLE
update upstream tf provider to v0.2.48

### PR DESCRIPTION
**Pull request by jubrad**
Notable new features
>Allow configuration of OIDC redirect URL by @benesch in https://github.com/frontegg/terraform-provider-frontegg/pull/114

https://github.com/frontegg/terraform-provider-frontegg/releases](https://github.com/frontegg/terraform-provider-frontegg/releases/tag/v0.2.48)

### CODE REVIEW

**Review by benesch — APPROVED** [review]

**Comment by benesch** [comment]
Thank you!

**Comment by benesch** [comment]
Sounds appealing, but also it's nice to be able to issue new versions of the Pulumi plugin without waiting for new upstream releases (e.g., a Pulumi specific bug that we fix independently of upstream). Defer to you!

**Comment by jubrad** [comment]
Wow based on our tag history we've been all over the place with this. 

Maybe for now it's better to `v0.2.28` and I'll create a follow on pr updating the maintainers MD with some suggestions.

**Pull request merged**
